code/modules/hydroponics/seeds.dm

```diff
@@ -1513,3 +1533,79 @@
 	water_consumption = 6
 	nutrient_consumption = 0.10
 	growth_stages = 4
+
+// Vox Food
+
+/datum/seed/mushroom/chicken
+	name = "chickenshroom"
+	seed_name = "chickenshroom"
+	display_name = "chicken of the stars"
+	products = list(/obj/item/weapon/reagent_containers/food/snacks/grown/mushroom/chickenshroom)
+	mutants = null
+	packet_icon = "mycelium-chickenshroom"
+	plant_dmi = 'icons/obj/hydroponics2.dmi'
+	plant_icon = "chickenshroom"
+	chems = list("nutriment" = list(2,10))
+	consume_gasses = list("nitrogen"=20) //Really likes its nitrogen. Planting on main station may mess with room air mix.
+
+	lifespan = 30
+	growth_stages = 3
+	maturation = 4
+	yield = 3
+	potency = 0
+
+/datum/seed/apple/wood
+	name = "woodapple"
+	seed_name = "woodapple"
+	display_name = "woodapple tree"
+	products = list(/obj/item/weapon/reagent_containers/food/snacks/grown/woodapple)
+	packet_icon = "seed-woodapple"
+	plant_dmi = 'icons/obj/hydroponics2.dmi'
+	plant_icon = "woodapple"
+	chems = list("sugar" = list(1,10))
+
+	growth_stages = 3
+	maturation = 4
+	yield = 4
+	potency = 20
+	ligneous = 1
+
```

Comment:
Presumably they would drop pods over and over

However since all seeds share a single global variable, a smart admin could intervene by opening any vine, navigating to the seed datum, and raising the production variable

Production governs reharvest time not growth time (which is maturation), for further discussion of the woodapple bug see vox hydroponics part ii